pages/profile.php

```diff
@@ -207,53 +211,56 @@ function userProfile() {
         echo "
 		<script>
 		var remainingtime = " . (59 - $time_since_last_regen) . ";
-
-		var health = {$player->health};
-		var max_health = {$player->max_health};
-
-		var chakra = {$player->chakra};
-		var max_chakra = {$player->max_chakra};
-
-		var stamina = {$player->stamina};
-		var max_stamina = {$player->max_stamina};
+        var statusBars = {
+            health: [{$player->health}, {$player->max_health}, 0],         
```

Comment:
Could we restructure the arrays here to objects so that we can name the keys and preserve the semantic meaning? (also, not totally sure what the third element is for. Reading below, is this for the post-regen amount?)
```suggestion
            health: {
              current: {$player->health}, 
              max: {$player->max_health}, 
              ???: 0
            },         
```

Reply:
Yep, it is for the amount after next regen. I'll move it into an object.